11/homework.py

```diff
@@ -0,0 +1,45 @@
+# TASK 1: Creating the decorator
+from datetime import datetime
+
+
+def my_decorator(func):
+    def inner(*args, **kwargs):
+        print(f'Function name is {func.__name__}, time of execution is {datetime.now()}')
+        func(*args, **kwargs)
```

Comment:
Use `return` statement here to be able to decorate functions that return some value